test/Sentry.Tests/Internals/MainSentryEventProcessorTests.cs

```diff
@@ -434,6 +434,24 @@ public void Process_CultureInfo_ValuesOnContext(Action<CultureInfo> setter, Func
             }
         }
 
+        [Fact]
+        public void Process_AppliesDefaultTags()
```

Comment:
Could we have one test that makes sure default tags don't override the event tag please?

Reply:
Looks good and I know we didnt have thst test to begin with but we should have it

Reply:
🔧